apps/sms/views/conversation/test/unit/conversation_test.js

```diff
@@ -6786,6 +6783,8 @@ suite('conversation.js >', function() {
         ConversationView.initRecipients();
         Navigation.isCurrentPanel.withArgs('composer').returns(true);
         this.sinon.stub(ConversationView.recipients, 'focus');
+        this.sinon.stub(Compose, 'focus');
+        this.sinon.stub(ConversationView, 'hasValidRecipients');
```

Comment:
I don't think it's a valid way to stub internal methods. Can you write tests that actually add drafts with/without recipients instead ?